### PR TITLE
update-readme

### PR DESCRIPTION
**Pull request by Irma12**

### CODE REVIEW

File: plugins/backstage-plugin-github-pull-requests/README.md [hunk 1]
```diff
@@ -2,20 +2,29 @@
 
 ![a list of pull requests in the GitHub Pull Requests](https://raw.githubusercontent.com/RoadieHQ/backstage-plugin-github-pull-requests/main/docs/list-of-pull-requests-and-stats-tab-view.png)
 
+
+## Notice
+
+If you have previously used this plugin you will be noticing some changes. In order to increase usability of our plugins and make testing and integration process straightforward and easy we are currently working on re-structuring and re-organising our plugins workflows and architecture.
```
Comment (dtuite) [inline plugins/backstage-plugin-github-pull-requests/README.md:8]: My suggested version basically says the same thing, just in a shorter fashion. It's also important to tell the users what is not changing, to proactively answer the top questions that may pop up in their minds.

```suggestion
In order to make testing and deployment of our plugins easier we are migrating all Roadie plugins to a monorepo at https://github.com/RoadieHQ/backstage-roadie-plugins.

The plugins will still be published to the same place on NPM and will have the same package names so nothing should change for consumers of these plugins.
```

File: plugins/backstage-plugin-github-pull-requests/README.md [hunk 2]
```diff
@@ -2,20 +2,29 @@
...
+
+We have decided to gather all of our plugins repositories in one place, so it is easier to find everything you need, which will boost the usage and hopefully give better overview of everything you can achieve with Backstage.
```
Comment (dtuite) [inline plugins/backstage-plugin-github-pull-requests/README.md:10]:
```suggestion
```

File: plugins/backstage-plugin-github-pull-requests/README.md [hunk 3]
```diff
@@ -2,20 +2,29 @@
...
+
+From now on, this plugin will be contained inside https://github.com/RoadieHQ/backstage-roadie-plugins repository, which will gather all of the plugins we, at Roadie, have worked on.
```
Comment (dtuite) [inline plugins/backstage-plugin-github-pull-requests/README.md:12]:
```suggestion
```

File: plugins/backstage-plugin-github-pull-requests/README.md [hunk 4]
```diff
@@ -2,20 +2,29 @@
...
+
+Updated instructions could be found below:
```
Comment (dtuite) [inline plugins/backstage-plugin-github-pull-requests/README.md:14]:
```suggestion
```

File: plugins/backstage-plugin-github-pull-requests/README.md [hunk 5]
```diff
@@ -2,20 +2,29 @@
 
 ![a list of pull requests in the GitHub Pull Requests](https://raw.githubusercontent.com/RoadieHQ/backstage-plugin-github-pull-requests/main/docs/list-of-pull-requests-and-stats-tab-view.png)
 
+
+## Notice
```
Comment (dtuite) [inline plugins/backstage-plugin-github-pull-requests/README.md:6]:
```suggestion
## Repository migration notice
```

I want to make this heading a little more alarming so people read it.

**Review by dtuite — CHANGES_REQUESTED** [review]

**Comment by martina-if** [comment]
related to #6

**Review by dtuite — APPROVED** [review]

**Pull request merged**